DMCompiler/Compiler/DMPreprocessor/DMPreprocessor.cs

```diff
@@ -747,7 +747,12 @@ private List<Token> GetLineOfTokens()
 
             int parenthesisNesting = 1;
             while(true) {
+
                 switch (parameterToken.Type) {
+                    case TokenType.Newline:
+                        currentParameter.Add(new Token(TokenType.DM_Preproc_LineSplice, "", parameterToken.Location, null));
+                        parameterToken = GetNextToken(true);
+                        continue;
```

Comment:
This does not compile in BYOND:
```
#define TEST(a,b) a+b

/proc/main()
    world.log << TEST(1, 2+
2)
```

This implies that you shouldn't just remove every newline you encounter. I think what BYOND is doing here is consuming an optional newline and/or whitespace after the comma separating the arguments.